src/rkopenmdao/error_estimator.py

```diff
@@ -0,0 +1,55 @@
+from abc import ABC, abstractmethod
+from numpy import linalg as LA
+import numpy as np
+
+
+def _convert_list_to_arr(arr):
+    if isinstance(arr, list):
+        arr = np.array(arr)
+    return arr
+
+
+class ErrorEstimator(ABC):
+    @abstractmethod
+    def __init__(self, quantity_metadata: dict):
+        pass
+    @abstractmethod
+    def __call__(self, u, embedded_u):
+        pass
+
+
+class SimpleErrorEstimator(ErrorEstimator):
+    """
+    Solves a norm for the difference of two vectors Delta = U - U_embedded
+    """
+    def __init__(self, ord=None):
+        self.ord = ord
+
+    def __call__(self, u, embedded_u):
+        u = _convert_list_to_arr(u)
```

Comment:
It also isn't wrong to just expect u and embedded_u to be numpy arrays. When it is coming from the RungeKuttaIntegrator, it will also be numpy arrays, and if the code is used for something else, it's on the users then to make sure numpy arrays get passed. I would just annotate in the signature that numpy arrays are expected.